internal/pkg/helper/tokens.go

```diff
@@ -0,0 +1,93 @@
+package helper
+
+import (
+	"crypto/rand"
+	"crypto/rsa"
+	"crypto/x509"
+	"encoding/pem"
+	"github.com/assetto-corsa-web/accweb/internal/pkg/cfg"
+	"github.com/sirupsen/logrus"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+)
+
+func GenerateTokenKeysIfNotPresent(config *cfg.Config) {
+	// Based on https://stackoverflow.com/questions/64104586/use-golang-to-get-rsa-key-the-same-way-openssl-genrsa
+	bitSize := 4096
+
+	logrus.Info("accweb: checking for secrets...")
+
+	// make sure files don't already exist
+	if Exists(config.Auth.PublicKeyPath) && Exists(config.Auth.PrivateKeyPath) {
+		logrus.WithField("file", config.Auth.PublicKeyPath).Info("Public/private keys already exists, not attempting regeneration")
+		return
+	} else if Exists(config.Auth.PublicKeyPath) {
+		logrus.WithField("file", config.Auth.PrivateKeyPath).Fatal("Private key file is missing (public key present)")
+	} else if Exists(config.Auth.PrivateKeyPath) {
+		logrus.WithField("file", config.Auth.PublicKeyPath).Fatal("Public key file is missing (private key present)")
```

Comment:
```suggestion
		logrus.WithField("file", config.Auth.PrivateKeyPath).Fatal("Private key file is missing")
```

Reply:
This is the opposite of the current logic, but I can see how it is confusing. It'd be better to give the user information back about what they can do to make it work instead of what is missing (since the actual change would be to remove the public key file). I'll look into it.

Reply:
I got it now